compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/error.todo-invalid-jsx-in-catch-in-outer-try-with-finally.expect.md

```diff
@@ -0,0 +1,56 @@
+
+## Input
+
+```javascript
+// @validateNoJSXInTryStatements
+import {identity} from 'shared-runtime';
+
+function Component(props) {
+  let el;
+  try {
+    let value;
+    try {
+      value = identity(props.foo);
+    } catch {
+      el = <div value={value} />;
+    }
+  } finally {
+    console.log(el);
+  }
+  return el;
+}
+
+```
+
+
+## Error
+
+```
+   4 | function Component(props) {
+   5 |   let el;
+>  6 |   try {
+     |   ^^^^^
+>  7 |     let value;
+     | ^^^^^^^^^^^^^^
+>  8 |     try {
+     | ^^^^^^^^^^^^^^
+>  9 |       value = identity(props.foo);
+     | ^^^^^^^^^^^^^^
+> 10 |     } catch {
+     | ^^^^^^^^^^^^^^
+> 11 |       el = <div value={value} />;
+     | ^^^^^^^^^^^^^^
+> 12 |     }
+     | ^^^^^^^^^^^^^^
+> 13 |   } finally {
+     | ^^^^^^^^^^^^^^
+> 14 |     console.log(el);
+     | ^^^^^^^^^^^^^^
+> 15 |   }
+     | ^^^^ Todo: (BuildHIR::lowerStatement) Handle TryStatement without a catch clause (6:15)
```

Comment:
i'm adding this pre-emptively so that when we support try w only a finally clause, we'll see this either continue to correctly fail or notice that the validation doesn't work and have a chance to fix it